### PR TITLE
Conditionally show edit or show-view for 'My application'-link

### PR DESCRIPTION
**Pull request by thesuss**
PT Story: https://www.pivotaltracker.com/story/show/136105855

Changes proposed in this pull request:
1. Fixed "wordo" in membership-application-show page
2. Conditionally displays edit/show depending on membership status 
    (if they type the link in, so be it for now, this is a quick fix and a known such)

Ready for review:
@AgileVentures/shf-project-team

### CODE REVIEW

File: app/views/application/_navigation_edit_my_application.html.haml [hunk 1]
```diff
@@ -1,2 +1,6 @@
-%li.menu-item
-  = link_to 'Min ansökan', edit_membership_application_path(membership_app)
+- if current_user.is_member?
```
Comment (weedySeaDragon) [inline app/views/application/_navigation_edit_my_application.html.haml]: Keep this DRY:  don't repeat the `%li.menu-item` or `link_to`:

````
%li.menu-item
  = link_to 'Min ansökan', current_user.is_member? ?  membership_application_path(membership_app) : edit_membership_application_path(membership_app)
````

**Review by weedySeaDragon — CHANGES_REQUESTED** [review]
Need to have tests that prove that the right menu item is displayed.

File: app/views/application/_navigation_edit_my_application.html.haml (hunk 1 above)
Comment (thesuss) [reply app/views/application/_navigation_edit_my_application.html.haml]: I don't know those shorthand things. I have probably learnt, but forgotten... -_-

Comment (weedySeaDragon) [reply app/views/application/_navigation_edit_my_application.html.haml]: it's fine to use a normal `if..then..else` if you want.  Important thing is to DRY it up.

Comment (thesuss) [reply app/views/application/_navigation_edit_my_application.html.haml]: Fixed it, but hound will say the line is too long..

**Review by weedySeaDragon — APPROVED** [review]

**Pull request merged**